ocaml/vncproxy/vncproxy.ml

```diff
@@ -73,7 +73,11 @@ let _ =
           ~http xml
     | host ->
         XMLRPC_protocol.rpc ~srcstr:"vncproxy" ~dststr:"xapi"
-          ~transport:(SSL (SSL.make ~use_fork_exec_helper:false (), host, 443))
+          ~transport:
+            (SSL
+               ( SSL.make ~verify_cert:None ~use_fork_exec_helper:false ()
```

Comment:
I would think this is a pool connection: we're connecting to the member of the pool that can offer the session to the VM.

Reply:
I've updated this.